cfme/tests/configure/test_access_control.py

```diff
@@ -1097,14 +1098,25 @@ def test_tenant_quota_input_validate(appliance):
 
 
 def test_delete_default_tenant(appliance):
+    """
+    Steps:
+
+    1. Login as an 'Administrator' user
+    2. Navigate to configuration > access control > tenants
+    3. Select default tenant('My Company') from tenants table
+    4. Delete using 'configuration > Delete selected items'
+    5. Check whether default tenant is deleted or not
+    """
     roottenant = appliance.collections.tenants.get_root_tenant()
     view = navigate_to(appliance.collections.tenants, 'All')
-    for row in view.table.rows():
-        if row.name.text == roottenant.name:
-            row[0].check()
-    msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
-    with pytest.raises(Exception, match=msg):
+    try:
+        for row in view.table.rows():
+            if row.name.text == roottenant.name:
+                row[0].check()
+        msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
         view.toolbar.configuration.item_select('Delete selected items', handle_alert=True)
```

Comment:
Optional:
As we have collection support, I would like to see `instantiated` object of `roottenant` and use `delete()` method of it.